daemon/internal/newrelic/integration/test.go

```diff
@@ -763,25 +765,15 @@ func (t *Test) comparePhpPackages(harvest *newrelic.Harvest) {
 				len(expectedPackages), len(actualPackages), expectedPackages, actualPackages))
 			return
 		}
-		for i, _ := range expectedPackages {
+		for i := range expectedPackages {
 			var matchingIdx int = -1
 			for j, pkg := range actualPackages {
-				//fmt.Printf("Comparing %s to %s\n", pkg.Name, expectedPackages[i].Name)
 				if pkg.Name == expectedPackages[i].Name {
-					//fmt.Printf("Match - index = %d\n", j)
```

Comment:
Were all of these comments left in previously for ease of future debugging efforts?

Reply:
Did the autoformatter remove them?